package.json

```diff
@@ -3,7 +3,11 @@
   "private": true,
   "kcd-workshop": {
     "title": "Testing Fundamentals",
-    "githubRoot": "https://github.com/epicweb-dev/testing-fundamentals"
+    "githubRoot": "https://github.com/epicweb-dev/testing-fundamentals",
+    "forms": {
+			"exercise": "https://docs.google.com/forms/d/e/1FAIpQLSf8xpntKQzPYYO9H1OvMcxEa224PqmhVeYfUSj_MotHy-fSUQ/viewform?usp=pp_url&entry.1709952199=Testing+Fundamentals",
```

Comment:
@kentcdodds, may I please ask you to double-check that these forms are OK?

Reply:
```suggestion
			"exercise": "https://docs.google.com/forms/d/e/1FAIpQLSf8xpntKQzPYYO9H1OvMcxEa224PqmhVeYfUSj_MotHy-fSUQ/viewform?usp=pp_url&entry.1709952199={workshopTitle}",
```

Reply:
You'll need to get a new link to get the exerciseTitle bit.